src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,159 @@
+    "WAVE is an extension available for both Chrome and Firefox that helps identify accessibility issues on web pages.",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
+  {
+    Question: "What is the primary function of a screen reader in web development?",
+    Answer: 
+    "Screen readers are used to convey the content of a webpage to users with visual impairments via text-to-speech, helping developers understand and create accessible web content.",
+    Distractor1: "It's a web design tool",
+    Distractor2: "It's a code editor",
+    Distractor3: "It's a graphics rendering software",
+    Explanation: 
+    "Screen readers assist users with visual impairments in accessing web content through spoken text.",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
+  {
+    Question: "Why is it important to add labels to form inputs like search forms?",
+    Answer: 
+      "Labels indicate the purpose of form inputs and make them accessible to screen reader users.",
+    Distractor1: "Labels improve website aesthetics.",
+    Distractor2: "Labels increase website loading speed.",
+    Distractor3: "Labels provide translations for form inputs.",
+    Explanation: 
+      "Labels associated with form inputs, such as search forms, describe the purpose of the input, making it accessible to screen reader users.",
+    Link: "https://www.freecodecamp.org/news/common-accessibility-errors-and-how-to-fix-them/"
+  },  
+  {
+    Question: "What are the four broad guidelines of web accessibility according to WCAG?",
+    Answer: 
+      "Perceivable, Operable, Understandable, Robust",
+    Distractor1: "Creative, Original, Unique, Dynamic",
+    Distractor2: "Left, Right, Up, Down",
+    Distractor3: "HTML, CSS, JavaScript, PHP",
+    Explanation: 
+      "The four broad guidelines of web accessibility are Perceivable, Operable, Understandable, and Robust, often referred to as WCAG.",
+    Link: "https://www.freecodecamp.org/news/what-is-website-accessibility-18ce00ec990f/"
+
+  },
+  {
+    Question: "Which screen reader is a Free & Open-source downloadable software for Windows OS?",
+    Answer: 
+      "NVDA (Non Visual Desktop Access)",
+    Distractor1: "JAWS (Job Access With Speech)",
+    Distractor2: "ChromeVox",
+    Distractor3: "a11yproject",
+    Explanation: 
+      "NVDA is a free screen reader for Windows, making it accessible to a wide audience.",
+      Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
+  {
+    Question: "What does the 'a11yproject' provide for web development, and what can you find on their site?",
+    Answer: 
+    "The 'a11yproject' offers practical guides, examples, how-tos, myths, tips, and recommended accessibility tools for implementing accessible features and components during web development. It is a community-driven project that can be contributed to on Github.",
```

Comment:
it is best to keep answers short.
the explanation field is the place for longer explanations

```suggestion
    "It provides practical guides, examples, how-tos, myths, tips, and recommended accessibility tools.",
```